Tests/Realm.Tests/RealmTest.cs

```diff
@@ -41,9 +41,9 @@ static RealmTest()
         {
 #if !UNITY
 #pragma warning disable CA1837 // Use Environment.ProcessId instead of Process.GetCurrentProcess().Id
-            InteropConfig.DefaultStorageFolder = Path.Combine(Path.GetTempPath(), $"rt-{System.Diagnostics.Process.GetCurrentProcess().Id}");
+            InteropConfig.SetDefaultStorageFolder(Path.Combine(Path.GetTempPath(), $"rt-{System.Diagnostics.Process.GetCurrentProcess().Id}"));
 #pragma warning restore CA1837 // Use Environment.ProcessId instead of Process.GetCurrentProcess().Id
-            Directory.CreateDirectory(InteropConfig.DefaultStorageFolder);
+            Directory.CreateDirectory(InteropConfig.GetDefaultStorageFolder("Could not determine a writable folder to store the Realm file. When constructing the RealmConfiguration, provide an absolute optionalPath where writes are allowed."));
```

Comment:
Same here - no error expected here, let's use a shorter message (and the remaining places)